app/views/shared/_breadcrumbs.html.erb

```diff
@@ -1,15 +1,16 @@
 <div class='al-search-breadcrumb'>
   <%= link_to t('arclight.routes.home'), root_path %>
-  <%= t('arclight.breadcrumb_separator') %>
+  <div class='bc_line'><%= t('arclight.breadcrumb_separator') %>
```

Comment:
Is it possible to use the `bc-line` instead and also fix the indention here?

Reply:
Fixed.